cmd/nerdctl/main.go

```diff
@@ -432,7 +441,7 @@ func AddStringFlag(cmd *cobra.Command, name string, aliases []string, value stri
 
 // AddPersistentStringFlag is similar to AddStringFlag but persistent.
 // See https://github.com/spf13/cobra/blob/master/user_guide.md#persistent-flags to learn what is "persistent".
-func AddPersistentStringFlag(cmd *cobra.Command, name string, aliases, nonPersistentAliases []string, value string, env, usage string) {
```

Comment:
It seems that we are not using `localAliases`

Reply:
we should keep it to differentiate the 3 types of aliases to avoid confusion.